ckanext/datarequests/controllers/controller_functions.py

```diff
@@ -130,10 +131,10 @@ def pager_url(state=None, sort=None, q=None, page=None):
     except ValueError as e:
         # This exception should only occur if the page value is not valid
         log.warn(e)
-        tk.abort(400, tk._('"page" parameter must be an integer'))
+        return tk.abort(400, tk._('"page" parameter must be an integer'))
```

Comment:
was this a bug?

Reply:
I reckon so. Under Pylons, it would have ended processing immediately, while under Flask - or with everything mocked out in a unit test - it would have proceeded.